.idea/2021-spring-cs160-team-EduAll.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
```

Comment:
There are some files that are IDE specific. Can you remove those, such as .idea....?